src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -49,10 +49,6 @@
     @FXML
     private StackPane personAndBidTabPanePlaceholder;
```

Comment:
We shud prob rename some of the variables to remove person and have a more general name like TabPanePlaceHolder